tessellation/src/basic_shapes.rs

```diff
@@ -803,26 +774,31 @@ pub fn stroke_circle<Output>(center: Point, radius: f32, tolerance: f32, output:
     }
 
     let angle = (0.0, 2.0 * PI);
+    let starting_point = center + vec2(1.0, 0.0) * radius;
 
     let arc_len = 0.5 * PI * radius;
```

Comment:
ok, changed that for every instance of  `arc_len`, since they are all based on the same procedure.